index.bs

```diff
@@ -3545,12 +3545,12 @@ Methods</h4>
 <dl dfn-type=method dfn-for="AudioParam">
 	: <dfn>cancelAndHoldAtTime(cancelTime)</dfn>
 	::
-		This is similar to {{AudioParam/cancelScheduledValues()}} in that it cancels all
-		scheduled parameter changes with times greater than or equal to
-		<code>cancelTime</code>. However, in addition, the automation
-		value that would have happened at <code>cancelTime</code> is
-		then proprogated for all future time until other automation
-		events are introduced.
+		This is similar to {{AudioParam/cancelScheduledValues()}} in that it can
+		remove events from the timeline after <code>cancelTime</code>.  However,
+		instead of just removing the events, the automation values leading to
+		<code>cancelTime</code> are not affected, and the value at
+		<code>cancelTime</code> is then propagated for all future time until other
+		automation events are introduced.
```

Comment:
> Does it mean automation events scheduled before `cancelTime` are not affected? I am not sure what "leading to" means here. This also suggests the events in the past won't be cleared out. Is it what we want? (This has performance implication.)

This is not observable from content, is it? It might be if there are curves in the past, but that's fixable.

It's not clear enough, I'll rephrase.

Reply:
FWIW, I agree with @hoch: the new phrasing is less clear than the old one. Especially the "automation values leading to cancelTime are not affected" part. I have no idea what that means. Also, the "can remove events from the timeline" is confusing. "Can" suggests it doesn't always remove scheduled events. Which raises the question: in what situations does it remove them?